pkg/zson/docs/spec.md

```diff
@@ -201,24 +197,24 @@ by a value encoding.  Here is a pseudo-grammar for value encodings:
         | <list> <elem> ;
 <terminal> := <char>*
 <char> := [^\n\\;[]] | <esc-sequence>
-<esc-sequence> := <javascript character escaping rules>
+<esc-sequence> := <JavaScript character escaping rules>
```

Comment:
Some comment as above: Is there a definitive URL we can point to for this?

Reply:
https://tc39.es/ecma262/#prod-EscapeSequence describes what can come after the `\` in JavaScript string literals, but I think we'll go with something much simpler .

Reply:
I will add this link below the table.